internal/kubernetes/operator/install_resources.go

```diff
@@ -323,6 +337,21 @@ func (ir *InstallResources) addDeployment(ctx context.Context, config map[string
 				obj.Spec.Template.Spec.Containers[0].Env[i] = env
 			}
 		}
+
+		if !deletionProtection {
+			obj.Spec.Template.Spec.Containers[0].Env = append(obj.Spec.Template.Spec.Containers[0].Env, corev1.EnvVar{
```

Comment:
Is there a preferable way to configure the manager? (command options vs environment vars)

Reply:
Just double checked, we can use both, but let's stick to flags. Thanks!